src/components/Sidebar/Participants/ParticipantsTab.vue

```diff
@@ -166,34 +166,50 @@ export default {
 		}, 250),
 
 		async fetchSearchResults() {
-			const response = await searchPossibleConversations(this.searchText)
-			this.searchResults = response.data.ocs.data
-			this.getParticipants()
-			this.contactsLoading = false
+			try {
+				const response = await searchPossibleConversations(this.searchText)
+				this.searchResults = response.data.ocs.data
+				this.getParticipants()
+				this.contactsLoading = false
+			} catch (exeption) {
+				console.debug(exeption)
```

Comment:
This is an error and is failing hard.
This should be at least a `console.error` :)

Do you do anything to warn the user?

Reply:
Ok for the error. No we don't warn for now

Reply:
Well you should, otherwise the user don't know what the hell is happening :p

Reply:
So what should we do here? cc @nickvergessen

Reply:
`OCP.Toast.error(t('spreed', 'An error occurred while performing the search'))` ?

Reply:
> ?

I'm asking you :sweat_smile:

Reply:
Done